cpp/src/sender.cpp

```diff
@@ -71,7 +72,11 @@ tracker sender::send(const message &message, const binary &tag) {
         pn_object(),
         pn_dtag((reinterpret_cast<const char *>(&tag[0])), tag.size()));
     std::vector<char> buf;
-    message.encode(buf);
+
+    Tracing& ot = Tracing::getTracing();
+    tracker track = make_wrapper<tracker>(dlv);
```

Comment:
This is now duplicated below in the return line (82). There you now should just `return tracker;`